docs/guide/essentials/config/environment-variables.md

```diff
@@ -42,6 +42,8 @@ WXT provides some custom environment variables based on the current command:
 | `import.meta.env.EDGE`             | `boolean` | Equivalent to `import.meta.env.BROWSER === "edge"`    |
 | `import.meta.env.OPERA`            | `boolean` | Equivalent to `import.meta.env.BROWSER === "opera"`   |
 
+The `BROWSER` can be a safer literal string union type of all browser names that you want to support, see [`targetBrowsers`](/api/reference/wxt/interfaces/InlineConfig#targetbrowsers) option.
```

Comment:
"safer literal string union type" is a mouthful lol. Let's go with something a little easier to understand:

```suggestion
You can set the [`targetBrowsers`](/api/reference/wxt/interfaces/InlineConfig#targetbrowsers) option to make the `BROWSER` variable a more specific type, like `"chrome" | "firefox"`.
```